### ISSUE TITLE
python-3.6: open_wr and unlink "access violations"

### ISSUE THREAD

**Issue by joe4379** [InBetweenNames/gentooLTO#93]
Like my 3.4 failure, I've got no idea where to start.  Thanks!

```
>>> Source compiled.
 * --------------------------- ACCESS VIOLATION SUMMARY ---------------------------
 * LOG FILE: "/var/log/sandbox/sandbox-10633.log"
 * 
VERSION 1.0
FORMAT: F - Function called
FORMAT: S - Access Status
FORMAT: P - Path as passed to function
FORMAT: A - Absolute Path (not canonical)
FORMAT: R - Canonical Path
FORMAT: C - Command Line

F: open_wr
S: deny
P: /usr/lib64/python3.6/site-packages/__pycache__/test.try
A: /usr/lib64/python3.6/site-packages/__pycache__/test.try
R: /usr/lib64/python3.6/site-packages/__pycache__/test.try
C: ./python -m test.regrtest -w -uall,-audio -x test_gdb test_multiprocessing test_subprocess test_tokenize test_signal test_faulthandler test_asyncio test_ctypes 

F: unlink
S: deny
P: /usr/lib64/python3.6/site-packages/__pycache__/test.try
A: /usr/lib64/python3.6/site-packages/__pycache__/test.try
R: /usr/lib64/python3.6/site-packages/__pycache__/test.try
C: ./python -m test.regrtest -w -uall,-audio -x test_gdb test_multiprocessing test_subprocess test_tokenize test_signal test_faulthandler test_asyncio test_ctypes 

F: open_wr
S: deny
P: /usr/lib64/python3.6/site-packages/__pycache__/test.try
A: /usr/lib64/python3.6/site-packages/__pycache__/test.try
R: /usr/lib64/python3.6/site-packages/__pycache__/test.try
C: ./python -m test.regrtest -w -uall,-audio -x test_gdb test_multiprocessing test_subprocess test_tokenize test_signal test_faulthandler test_asyncio test_ctypes 

F: unlink
S: deny
P: /usr/lib64/python3.6/site-packages/__pycache__/test.try
A: /usr/lib64/python3.6/site-packages/__pycache__/test.try
R: /usr/lib64/python3.6/site-packages/__pycache__/test.try
C: ./python -m test.regrtest -w -uall,-audio -x test_gdb test_multiprocessing test_subprocess test_tokenize test_signal test_faulthandler test_asyncio test_ctypes 
 * --------------------------------------------------------------------------------
`
```

**Comment by InBetweenNames** [comment]
Closed #92

**Comment by joe4379** [comment]
Thanks for the help and the project
